packages/loader/container-loader/src/containerStorageAdapter.ts

```diff
@@ -89,8 +87,7 @@ export class ContainerStorageAdapter implements IDocumentStorageService, IDispos
 			this.logger,
 		));
 
-		this._summarizeProtocolTree =
-			this._summarizeProtocolTree ?? service.policies?.summarizeProtocolTree;
+		this._summarizeProtocolTree = service.policies?.summarizeProtocolTree;
```

Comment:
It looks like this can still be disabled via policies. Is this always true for ODSP now?

Reply:
Also, since this change is in the loader layer, can't old loaders still do 2-commit summaries?

Reply:
As long as we can properly process the 2-commit summaries, we should be fine

Reply:
Should this default to true?

Reply:
Note: there can still be some weird loader-driver compat boundary issues. There are some clients that load with an older driver and a newer loader. As long as all the current drivers in production have this policy enabled I have no problems with this change.

Reply:
Correct, old clients could still submit 2-commit summaries, and latest client should handle it just fine.
Policy object comes from the driver. Yes, someone could make a mistake and change ODSP driver not to use single commit. At some point that would stop working as ODSP will not accept 2-comit world. We could make this policy come from service (not from our code), but I think it does not matter, we are moving toward getting to single-comit across all end-points.